xbmc/video/VideoDatabase.cpp

```diff
@@ -4566,6 +4589,13 @@ void CVideoDatabase::UpdateTables(int iVersion)
       }
     }
   }
+  if (iVersion < 94)
```

Comment:
If you change something in the database code that either requires manual update code or re-creation of the views, triggers, indices, ... you always need to bump the schema version. It already is at `94` so not sure how this is working for you (maybe because you directly updated from `93` or from no previous version at all). Anyone that already has a `MyVideos94.db` won't be getting these manual update instructions and than all queries to these tables will break.

You need to use `95` and also bump the schema version to that.